ale/base/data_naif.py

```diff
@@ -21,9 +23,17 @@ def __exit__(self, exc_type, exc_val, exc_tb):
         this is done, the object is out of scope and the
         kernels can be unloaded.
         """
-        spice.unload(self.metakernel)
+        if self.kernels:
+            [spice.unload(k) for k in self.kernels]
+        else:
+            spice.unload(self.metakernel)
 
     @property
+    def kernels(self):
+        if not hasattr(self, '_kernels'):
+            self._kernels =  self._props.get('kernels', None)
+        return self._kernels
```

Comment:
Removing the metakernel property and having everything go through the kernels property would make the loading and unloading code simpler. ALE is only concerned with throwing these values into furnsh and unload so passing the metakernel though the kernels property as just a single element list will be functionally the same.

Reply:
As of now, it's hard to separate kernels and metakernels because of the mission-specific configs. When we figure out the load interface, we will make it easier to classify Driver's mission from the modules and move to one metakernel definition so we should be able to consolidate the two then.